tests/server_notices/test_server_notices_manager.py

```diff
@@ -0,0 +1,197 @@
+# Copyright 2018, 2019 New Vector Ltd
```

Comment:
```suggestion
# Copyright 2022 The Matrix.org Foundation C.I.C.
```

Or your own name.

Reply:
I removed this test file and moved the tests to `tests/rest/admin/test_server_notice`.